### PR TITLE
api: Support include tombstone option for store.GetAllLimit

### PR DESCRIPTION
**Pull request by Yisaer**
### What problem does this PR solve?
Close https://github.com/pingcap/pd/issues/2714

### What is changed and how it works?
Add `includeTombstone` to dedicate whether the server need to return the store limit information include tombstone stores.

### Check List

Tests

- Unit test

Related changes

- PR to update [`pingcap/docs`](https://github.com/pingcap/docs)/[`pingcap/docs-cn`](https://github.com/pingcap/docs-cn):
- PR to update [`pingcap/tidb-ansible`](https://github.com/pingcap/tidb-ansible):
- Need to cherry-pick to the release branch

### CODE REVIEW

File: server/api/store.go [hunk 1]
```diff
@@ -464,12 +464,35 @@ func (h *storesHandler) SetAllLimit(w http.ResponseWriter, r *http.Request) {
...
 // @Tags store
 // @Summary Get limit of all stores in the cluster.
+// @Param includeTombstone query bool false "include Tombstone" default(true)
 // @Produce json
 // @Success 200 {object} string
 // @Failure 500 {string} string "PD server failed to proceed the request."
 // @Router /stores/limit [get]
 func (h *storesHandler) GetAllLimit(w http.ResponseWriter, r *http.Request) {
 	limits := h.GetScheduleConfig().StoreLimit
+	includeTombstone := true
```
Comment (rleungx) [inline server/api/store.go:474]: I think the default value should be false?

File: server/api/store.go [hunk 2]
```diff
@@ -464,12 +464,35 @@ func (h *storesHandler) SetAllLimit(w http.ResponseWriter, r *http.Request) {
...
+	var err error
+	if includeStr := r.URL.Query().Get("includeTombstone"); includeStr != "" {
```
Comment (rleungx) [inline server/api/store.go:476]: How about
```suggestion
	if includeStr := r.URL.Query().Get("include_tombstone"); includeStr != "" {
```

File: server/api/store.go (hunk 1 above)
Comment (lhy1024) [reply server/api/store.go:474]: I also think false is better

**Pull request merged**